api.swagger.yaml

```diff
@@ -1709,6 +1789,59 @@ paths:
         '422':
           description: JSON schema validation failure
       description: Report a post as the current user.
+  '/manuals':
+    get:
+      description: Get all the manual (root) pages
+      tags:
+        - manuals
+      parameters:
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: Returns all the root manual pages
```

Comment:
I would go with something like 'Returns a list of manuals'

root manual pages could be confused with the manual page content type. Which we aren't returning here.